cypress/integration/cbl/admin/tasks-manager.js

```diff
@@ -554,6 +539,12 @@ describe('CBL / Admin / Tasks Manager', () => {
                         items.push($div.text().trim());
                     })
             })
-        return items;
+            .then(() => {
+                if (reverse) {
+                    cy.expect(items, `Column [index: ${col}]: items are sorted in reverse alphabetical order.`).to.deep.equal(items.slice().sort().reverse());
+                } else {
+                    cy.expect(items, `Column [index: ${col}]: items are sorted in alphabetical order.`).to.deep.equal(items.slice().sort());
+                }
+            })
```

Comment:
I've never seen the `.then(() => { cy.expect(...) } ` pattern...seems like it _should_ work

Typically what I do in these cases is pass a function to `.should()` and use the bare `expect` function. I know that when you do that, the `.should()` wrapper catches the exceptions thrown by any failing assertions within and retries the whole `.should()` block up to the allowed timeout. 

The current implementation feels a bit odd with the shared state between the `.each()` and `.then()`. You could do all of this in a single `.should()` method which receives the subject element(s) already wrapped in jquery

I think you could just do something like:

```javascript
cy.get('.x-grid-row', { log: false })
            .should($rows => {
                expect($rows.length).to.be(20);
                const items = $rows.map($row => $row.find('.x-grid-cell-inner').text())
                expect(items)...;
            });